### ISSUE TITLE
Problems with Charm of weightless flow

### ISSUE THREAD

**Issue by imatrix127** [masik16u/Mythic-Charms#15]
### Discussed in https://github.com/masik16u/Mythic-Charms/discussions/14

<div type='discussions-op-text'>

<sup>Originally posted by **imatrix127** February  1, 2024</sup>
The charm is made and in my charm slot, but I jump or walk off a mountain and it doesn't work. Playing on better MC version 1.20.1 this did not work in singleplayer or multiplayer. Tried changing the mod version twice with no effect. any suggestions would be helpful please and thank you.</div>

**Comment by masik16u** [comment]
As I understood, it doesn't do anything?

**Comment by imatrix127** [comment]
correct here is video to show that the other charms do work just not weightless flow
sorry for low quality bitrate, had to compress to get to under 10MB for github



https://github.com/masik16u/Mythic-Charms/assets/29157455/35e40aec-7e51-4d75-a4e7-e6fec440d910

**Comment by masik16u** [comment]
I see, I will try to do something but can't promise anything. I tried it with only Mythic Charms mod and it works fine, so another mod may be the problem

**Comment by imatrix127** [comment]
ok, I will try to see if mod conflicts tomorrow, it's 3 am for me. thanks for the quick response, good luck hopefully you find the issue.

**Comment by imatrix127** [comment]
heres what ive done so far, to check mod compatibility I tried doing Binary sort method of disabling sections of mods at a time while having mythic charms enabled at the same time, then each time jumped into a singleplayer world and tried the charm. no effect to the charms with disabling any of the mods. i did notice however that the mods owo-lib and moonlight Lib had to be enabled to have mythic charm enabled but i dont know how that effects mods. I also checked latest.log file to see if any conflicts popped up in log but i dont see anything but i also dont really know what im looking for. I also updated all mods which actually hurt me because not all mods should be updated had to repair profile. sorry i couldnt be more helpful. By the way this is better Minecraft FABRIC version BMC2.

**Comment by imatrix127** [comment]
ok, new info from this same thing, with the weightless charm enabled it doesn't work but then I disconnect and reconnect to this server the weightless effect stays on permanently and doesn't let me jump or shift to get out. the effect lasts until I die.  This is not the case for singleplayer, weightless does not work at all still on singleplayer.

**Comment by masik16u** [comment]
So you tried launching only mythic charms and charm still doesn't work? Can you tell me what version of mythic charms you use? I don't know why it doesn't work without moonlight lib but owo-lib is required for mythic charms.

**Comment by imatrix127** [comment]
looks like 0.6.0 im just using default stuff for better Minecraft BMC2. by itself works. and I was wrong about moonlight lib sorry

**Comment by masik16u** [comment]
Ok, so I may have a solution, maybe you can test it?

**Comment by imatrix127** [comment]
ok so i actually think i found the mod conflict. the mod is called BadOptimizations(very fitting for the problem) by thosea, all other mods on and this one disabled and it works. turn it on and it doesn't work. Voila

**Comment by masik16u** [comment]
"Most optimization mods focus on rendering performance and optimizing Minecraft's render engine" (from mod description), what does this have to do with entity gravity D:

**Comment by masik16u** [comment]
Anyway, thanks for the information

**Comment by imatrix127** [comment]
still have issue with servers though, if you disconnect from server with the charm on you will stay in a perpetual gravityless-ness.  something to look at, thanks for being a great mod maker!

**Comment by masik16u** [comment]
Yeah, that thing with "you will stay in a perpetual gravityless-ness" should be fixed in the next update

**Comment by masik16u** [comment]
I released a new version of a mod, please let me know if the problem is resolved

**Comment by masik16u** [comment]
No further confirmation, so closing the issue as completed
